src/components/home/homeSearch/homeSearch.module.scss

```diff
@@ -1,27 +1,13 @@
 .searchField {
-  height: auto;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 30px 0;
+  width: 500px;
```

Comment:
Mobile devices has screenW < 500px so better to use max-width: 500px to allow reduce content according to smaller screenW

Reply:
In 99% strict width is bad practice